app/helpers/pipeline_runs_helper.rb

```diff
@@ -65,10 +65,15 @@ def aegea_batch_submit_command(base_command,
                                  docker_image: "idseq_dag",
                                  sample_id: 0,
                                  stage_name: "misc")
+    # TODO: environment values once we use the new pipeline infra
```

Comment:
Should we refer to a Jira ticket here?

Reply:
I am not sure which one it is. This will eventually need to be removed completely. @kislyuk do we have a ticket to capture the decoupling work?

Reply:
actually maybe I should just remove the TODO since instead of fixing it this section will just be deleted

Reply:
Should have a ticket anyway... If you find an existing one, great. Otherwise create a new one.